source/scripting/Ped.cs

```diff
@@ -1555,6 +1555,10 @@ public void OpenParachute()
 		{
 			Function.Call(Hash.FORCE_PED_TO_OPEN_PARACHUTE, Handle);
 		}
+		public void ParachuteTo(Vector3 location)
+		{
+		    	Function.Call(Hash.SET_PARACHUTE_TASK_TARGET, Handle, location.X, location.Y, location.Z);
```

Comment:
It does nothing but call the exact same native, so I don't see why it would behave any different?